src/LiveComponent/tests/Fixtures/Form/FormWithManyDifferentFieldsType.php

```diff
@@ -41,6 +43,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'foo' => 1,
                     'bar' => 2,
                 ],
+                'required' => false,
+            ])
+            ->add('choice_required', ChoiceType::class, [
+                'choices' => [
+                    'foo' => 1,
+                    'bar' => 2,
+                ],
             ])
             ->add('choice_expanded', ChoiceType::class, [
```

Comment:
```suggestion
                'required' => false,
            ])
            ->add('choice_required_with_placeholder', ChoiceType::class, [
                'choices' => [
                    'bar' => 2,
                    'foo' => 1,
                ],
                'required' => true,
                'placeholder' => 'foo',
            ])
            ->add('choice_required_without_placeholder', ChoiceType::class, [
                'choices' => [
                    'bar' => 2,
                    'foo' => 1,
                ],
                'required' => true,
                'placeholder' => false,
            ])
            ->add('choice_expanded', ChoiceType::class, [
```

I added a test to distinct the two scenarios